website/client/src/components/payments/selectUserModal.vue

```diff
@@ -122,181 +116,176 @@
       class="g1g1-fine-print text-center pt-3"
     >
       <strong>
-        {{ $t ('howItWorks') }}
```

Comment:
You usually advise me to use `v-once` with elements containing `$t` strings, should these be -onced?

Reply:
most of those changes in this file are from the automatic formatter xD - but yeah these definitely need once on it